When Building in Release mode Plugin.Admob version 3.0.2 causes Android java error

If I use <PackageReference Include="Plugin.AdMob" Version="3.0.2" />   I will get these java errors when compiling or building for release even if I am building for iOS. However, if I change to a lower version such as <PackageReference Include="Plugin.AdMob" Version="2.3.2-beta.4" />
I do not get these errors.
Currently I cannot publish any Android app with version 3.0.2 because of this error. 
Any help please?!

2>MSBUILD : java.exe error JAVA0000: Type androidx.compose.runtime.StableMarker is defined multiple times: C:\Ng\xamarin.androidx.compose.runtime.annotation.android\1.10.0.1\buildTransitive\net10.0-android36.0\..\..\aar\runtime-annotation-android.aar:classes.jar:androidx/compose/runtime/StableMarker.class, C:\Ng\xamarin.androidx.compose.runtime.annotation.jvm\1.10.0.1\buildTransitive\net10.0-android36.0\..\..\jar\runtime-annotation-jvm.jar:androidx/compose/runtime/StableMarker.class
2>MSBUILD : java.exe error JAVA0000: Compilation failed
2>MSBUILD : java.exe error JAVA0000: java.lang.RuntimeException: com.android.tools.r8.CompilationFailedException: Compilation failed to complete, origin: C:\Ng\xamarin.androidx.compose.runtime.annotation.android\1.10.0.1\buildTransitive\net10.0-android36.0\..\..\aar\runtime-annotation-android.aar:classes.jar
2>MSBUILD : java.exe error JAVA0000: androidx/compose/runtime/StableMarker.class
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.xv.a(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:131)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.R8.main(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:5)
2>MSBUILD : java.exe error JAVA0000: Caused by: com.android.tools.r8.CompilationFailedException: Compilation failed to complete, origin: C:\Ng\xamarin.androidx.compose.runtime.annotation.android\1.10.0.1\buildTransitive\net10.0-android36.0\..\..\aar\runtime-annotation-android.aar:classes.jar:androidx/compose/runtime/StableMarker.class
2>MSBUILD : java.exe error JAVA0000: 	at Version.fakeStackEntry(Version_8.11.18.java:0)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.P.a(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:1)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.xv.a(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:82)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.xv.a(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:32)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.xv.a(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:31)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.R8.b(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:62)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.R8.a(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:1658)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.xv.a(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:118)
2>MSBUILD : java.exe error JAVA0000: 	... 1 more
2>MSBUILD : java.exe error JAVA0000: Caused by: com.android.tools.r8.internal.g: Type androidx.compose.runtime.StableMarker is defined multiple times: C:\Ng\xamarin.androidx.compose.runtime.annotation.android\1.10.0.1\buildTransitive\net10.0-android36.0\..\..\aar\runtime-annotation-android.aar:classes.jar:androidx/compose/runtime/StableMarker.class, C:\Ng\xamarin.androidx.compose.runtime.annotation.jvm\1.10.0.1\buildTransitive\net10.0-android36.0\..\..\jar\runtime-annotation-jvm.jar:androidx/compose/runtime/StableMarker.class
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.yg0.a(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:21)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.O80.a(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:54)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.O80.a(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:10)
2>MSBUILD : java.exe error JAVA0000: 	at java.base/java.util.concurrent.ConcurrentHashMap.merge(ConcurrentHashMap.java:2056)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.O80.a(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:6)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.graph.z4$a.c(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:6)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.dex.c.a(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:56)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.dex.c.a(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:12)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.R8.a(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:33)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.R8.c(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:8)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.R8.a(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:1657)
2>MSBUILD : java.exe error JAVA0000: 	at com.android.tools.r8.internal.xv.a(R8_8.11.18_94248c9a6fd57686ded562f231d18f14e8923ad1fe977e99b974a9bf901c8a48:28)
2>MSBUILD : java.exe error JAVA0000: 	... 4 more
2>MSBUILD : java.exe error JAVA0000: Directory 'obj\Release\net10.0-android\lp\290' is from 'Xamarin.KotlinX.Serialization.Core.Jvm.aar'.

--- a/docs/Troubleshooting-Android.md
+++ b/docs/Troubleshooting-Android.md
@@ -14,13 +14,23 @@
   - Clean the solution
   - Delete `bin` and `obj`
   - Rebuild
-- **If you use .NET 10 and hit duplicate Compose annotation classes** (see closed issue #67): this is fixed in .NET MAUI 10.0.80 — update `Microsoft.Maui.Controls` to 10.0.80 or later and remove the workaround. On older MAUI 10 versions, exclude the JVM variant from your app project:
+- **If you hit duplicate Compose annotation classes on .NET 10** (issues #67 and #80, e.g. `Type androidx.compose.runtime.StableMarker is defined multiple times`): the root cause is `Xamarin.GooglePlayServices.Ads.Lite` 124.0.0.4 (shipped with plugin 3.0.2 and earlier) transitively pulling both the `.Android` and `.Jvm` variants of `Xamarin.AndroidX.Compose.Runtime.Annotation`. This is fixed in `Xamarin.GooglePlayServices.Ads.Lite` 124.0.0.5+, which plugin versions after 3.0.2 reference. On plugin 3.0.2 or earlier, either update the Ads SDK directly in your app project (verified fix):
+
+```
+<ItemGroup Condition="'$(TargetFramework)' == 'net10.0-android'">
+	<PackageReference Include="Xamarin.GooglePlayServices.Ads.Lite" Version="124.0.0.6" />
+</ItemGroup>
+```
+
+  or exclude the JVM variant (also verified):
 
 ```
 <ItemGroup Condition="'$(TargetFramework)' == 'net10.0-android'">
 	<PackageReference Include="Xamarin.AndroidX.Compose.Runtime.Annotation.Jvm" Version="1.10.0.1" ExcludeAssets="all" />
 </ItemGroup>
 ```
+
+  Note: updating your app's `Microsoft.Maui.Controls` version does **not** fix this — the duplicate variants come from the Ads SDK's dependency graph, not MAUI's.
 
 - **If you use Firebase and hit duplicate Google measurement classes** (see issues #29 and #55, e.g. `Type com.google.android.gms.internal.measurement.zzbm is defined multiple times`): the Google "measurement" classes ship in both `Xamarin.GooglePlayServices.Measurement.*` (pulled in by this plugin via `Xamarin.GooglePlayServices.Ads.Lite`) and `Xamarin.Firebase.Analytics` (pulled in by Plugin.Firebase). The build only works when the whole family resolves to the same Google release train. Align them by referencing `Xamarin.Firebase.Analytics` directly in your app at the version matching the resolved `Xamarin.GooglePlayServices.Measurement.Base` (check `obj/project.assets.json`). Verified with Plugin.Firebase 4.2.1 and Xamarin.GooglePlayServices.Ads.Lite 124.0.0.6:
 
@@ -36,6 +46,7 @@
 
 - [`#29`](https://github.com/marius-bughiu/Plugin.AdMob/issues/29) (duplicate measurement classes when combined with Firebase)
 - [`#67`](https://github.com/marius-bughiu/Plugin.AdMob/issues/67) (duplicate Compose runtime annotation classes on .NET 10)
+- [`#80`](https://github.com/marius-bughiu/Plugin.AdMob/issues/80) (duplicate Compose annotation classes in Release builds on plugin 3.0.2)
 
 ## AndroidManifest namespace warnings on .NET 10 / API 35+
 

